content/docs/forwarding-refs.md

```diff
@@ -1,76 +1,76 @@
 ---
 id: forwarding-refs
-title: Forwarding Refs
+title: Reenviament de Refs
```

Comment:
```suggestion
title: Passar Refs
```

Reply:
In my opinion, this better reflects the meaning of the concept that's explained in this section.